saray/weather-card/index.html

```diff
@@ -7,9 +7,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="stylesheet.css" />
     <link href="https://fonts.googleapis.com/css?family=Montserrat:400,300,500i,600,700" rel="stylesheet">
-    <script src="main.js"></script>
 </head>
-<body>
+<body class="container">
```

Comment:
Remember that you need to use the same convention to the names of all classes in the document. It should create collisions.

Reply:
since this class is only with the purpose of show he the wheater-container adjust to its container, and the class is in the body .. we can let it  pass :)